apps/studio/localStores/storageExplorer/StorageExplorerStore.ts

```diff
@@ -389,20 +389,23 @@ class StorageExplorerStore {
       return filePreview.url
     } else {
       const signedUrl = await this.fetchFilePreview(file.name, expiresIn)
-      const formattedUrl = new URL(signedUrl)
-      formattedUrl.searchParams.set('t', new Date().toISOString())
-      const fileUrl = formattedUrl.toString()
-
-      // Also save it to cache
-      const fileCache = {
-        id: file.id,
-        url: fileUrl,
-        expiresIn: DEFAULT_EXPIRY,
-        fetchedAt: Date.now(),
-      }
-      this.addFileToPreviewCache(fileCache)
+      if (signedUrl) {
+        const formattedUrl = new URL(signedUrl)
```

Comment:
For this one instead of checking that there is a `signedUrl` you could also wrap the `new URL()` call in a `try/catch` and then just return `''` if it fails? wdyt?

Reply:
ahh good point